packages/devtools/test/composite.test.ts

```diff
@@ -1,4 +1,14 @@
+import { jest } from '@jest/globals'
```

Comment:
We shouldn't need this whole setup here, it's already covered by using the `glaze` jest environment, see https://github.com/ceramicstudio/js-glaze/blob/main/packages/devtools/test/datamodel.test.ts for example.